### PR TITLE
OCPBUGSM-19651: sending event about cluster installation started

### PR DESCRIPTION
**Pull request by osherdp**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -999,11 +999,14 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
...
-			//send metric when the installation process has been started
-			b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		if err = b.db.Transaction(cInstaller.install); err != nil {
+			return
 		}
+
+		// send metric and event that installation process has been started
+		b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		msg := fmt.Sprintf("Updated status of cluster %s to installing", cluster.ID)
+		b.eventsHandler.AddEvent(ctx, *cluster.ID, nil, models.EventSeverityInfo, msg, time.Now())
```
Comment (avishayt) [inline internal/bminventory/inventory.go:1009]: No need for msg variable, just add the message to the AddEvent call

Comment (osherdp) [reply internal/bminventory/inventory.go]: done

File: internal/bminventory/inventory.go [hunk 2]
```diff
@@ -999,11 +999,15 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
...
-			b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		if err = b.db.Transaction(cInstaller.install); err != nil {
+			return
 		}
+
+		// send metric and event that installation process has been started
+		b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		b.eventsHandler.AddEvent(
+			ctx, *cluster.ID, nil, models.EventSeverityInfo,
+			fmt.Sprintf("Updated status of cluster %s to installing", cluster.ID), time.Now())
```
Comment (rollandf) [inline internal/bminventory/inventory.go:1010]: Why not use cluster name instead of ID?

Comment (osherdp) [reply internal/bminventory/inventory.go:1010]: You're right
If anything, maybe we shouldn't even need the name, as it's already bounded to a specific cluster view

Comment (avishayt) [reply internal/bminventory/inventory.go:1010]: No need for name or ID

File: internal/bminventory/inventory.go [hunk 3]
```diff
@@ -993,7 +993,7 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
 
 		defer func() {
 			if err != nil {
-				log.WithError(err).Warn("Cluster install")
+				log.WithError(err).Warn("Cluster installallation initialization failed")
```
Comment (rollandf) [inline internal/bminventory/inventory.go:996]: typo in `installallation`

Comment (osherdp) [reply internal/bminventory/inventory.go]: you're right
fixed

**Review by avishayt — APPROVED** [review]

**Pull request merged**